logic_functions.py

```diff
@@ -21,9 +21,9 @@ def get_fresh_board():
 
 
 
-def get_next_board(squares, i, x_is_next):
-    if squares[i] == "X":
-        x_is_next = "O"
+def get_next_board(squaresB, i, x_is_next):
+    if x_is_next == "X":
```

Comment:
I think you need to study the code you are porting from, what do you expect `x_is_next` to be here?

Reply:
so I was thinking that is was "X" or "O" but now that i relook at it I think that x_is_next is either True or False